nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -94,6 +95,14 @@ def wrapped(*args, **kwargs):
                     if isinstance(curr_module, _NNCFModuleMixin):
                         ignored_algos = deepcopy(curr_module.ignored_algorithms)
 
+                if op_name in ["reshape", "view", "flatten", "squeeze", "unsqueeze"]:
+                    input_tensor = args[0]
+                    output_shape = args[1:]
+
+                    layer_attrs = ReshapeLayerAttributes(input_tensor.shape,
```

Comment:
@vshampor I believe there is more obvious way to do this. Could you please help me to improve this?

Reply:
I'll move this section into dynamic_graph -> graph converting stage, as we discussed